cache.go

```diff
@@ -73,3 +79,27 @@ func (e *Exporter) addOrReuseCacheLayer(cache Cache, layerDir layerDir, previous
 	e.Logger.Debugf("Layer '%s' SHA: %s\n", layer.ID, layer.Digest)
 	return layer.Digest, cache.AddLayerFile(layer.TarPath, layer.Digest)
 }
+
+func (e *Exporter) addSBOMCacheLayer(layersDir string, cacheStore Cache, origMetadata platform.CacheMetadata, meta *platform.CacheMetadata) error {
+	// Look for sBOM cache layer
+	sbomLayer, err := readLayersConfigSBOM(layersDir, "cache", e.Logger)
```

Comment:
```suggestion
	sbomDir, err := readLayersConfigSBOM(layersDir, "cache", e.Logger)
```
(`Layer` makes me think we're looking for a tarball)

Reply:
_Fixed in latest commit._